src/Faker/Provider/th_TH/Person.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+namespace Faker\Provider\th_TH;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{titleMale}} {{firstNameMale}} {{lastName}}',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}}',
+    );
+
+    /**
+     * Male first names below are extracted from soap drama characters.
+     * @see https://th.wikipedia.org/wiki/%E0%B8%A3%E0%B8%B2%E0%B8%A2%E0%B8%8A%E0%B8%B7%E0%B9%88%E0%B8%AD%E0%B8%A5%E0%B8%B0%E0%B8%84%E0%B8%A3%E0%B9%82%E0%B8%97%E0%B8%A3%E0%B8%97%E0%B8%B1%E0%B8%A8%E0%B8%99%E0%B9%8C%E0%B9%84%E0%B8%97%E0%B8%A2
+     */
+    protected static $firstNameMale = array(
```

Comment:
@fzaninotto Thanks for the input. Will update accordingly.

Side question: I didn't see any guidelines on this (e.g. minimum items for a dictionary), at least not in CONTRIBUTING.md. Should this be provided somewhere? I can help develop one if needed.